modules/metastore/src/Storage/Data.php

```diff
@@ -456,21 +456,20 @@ private function filterHtml(mixed $input, string $parent = 'dataset') {
    *
    * @return string
    *   Filtered string.
-   *
-   * @codeCoverageIgnore
    */
   private function htmlPurifier(string $input) {
     // Initialize HTML Purifier cache config settings array.
     $config = [];
 
     // Determine path to tmp directory.
+    // @todo Inject this service.
```

Comment:
Did you mean this as a reminder to yourself for this ticket or for future selves?

Reply:
Our future selves. 😄

Reply:
I always have this same dilemma myself.  When I am in the middle of building something I leave myself a trail of @todos .  I have always thought there should be two levels.  @todo and @now-dammit